src/test/kotlin/org/rust/lang/core/resolve/RsMultiResolveTest.kt

```diff
@@ -21,6 +21,29 @@ class RsMultiResolveTest : RsResolveTestBase() {
         }
     """)
 
+    //FIXME: should resolve to a single mut method!
+    fun `test auto deref and deref mut`() = doTest("""
+        #[lang = "deref"]
+        trait Deref { type Target; }
+        #[lang = "deref_mut"]
+        trait DerefMut { type Target; }
```

Comment:
Hm, there's no  `Target` type in `DerefMut` in Rust. Instead, it extends `Deref` (`DerefMut: Deref`) and uses its `Target`.